db/views/hmis_households_v01.sql

```diff
@@ -0,0 +1,30 @@
+-- To update, run rails generate scenic:view hmis_households --replace
+
+SELECT
+  "Enrollment"."HouseholdID" AS id,
```

Comment:
primary ID should probably be `CONCAT(MAX("Enrollment"."HouseholdID"), MAX("Enrollment"."ProjectID"), "data_source_id)`